src/Broker.php

```diff
@@ -173,7 +173,7 @@ public function getConnect(string $key, string $type, bool $modeSync = false): ?
         $port = null;
 
         if (isset($this->brokers[$key])) {
-            $hostname      = $this->brokers[$key];
+            $hostname = $this->brokers[$key];
```

Comment:
I only changed these because they were triggering the Code Quality tests to fail.